tests/std/tests/Dev10_860410_bitset_ctors/test.cpp

```diff
@@ -12,6 +12,25 @@ using namespace std;
 
 #define STATIC_ASSERT(...) static_assert(__VA_ARGS__, #__VA_ARGS__)
 
+// Also test LWG-4140 "Useless default constructors for bit reference types" for bitset<N>::reference.
+namespace lwg_4140 {
+    struct default_constructible_type {};
+
+    void test_default_constructor(default_constructible_type) {}

```

Comment:
No change requested, but this test (ditto for `Dev11_0437519_container_requirements`) seems a bit obfuscated when we could simply `static_assert(!is_default_constructible_v</* ... */>);` each of these `reference` types. Am I missing some subtlety here?

Reply:
`is_default_constructible_v` is already `false` for these types. The existence of private default ctors can be observed in a tricky way, see LWG-3160.